pkg/pillar/types/zedagenttypes.go

```diff
@@ -539,3 +540,8 @@ type DeviceOpsCmd struct {
 	DesiredState bool
 	OpsTime      string
 }
+
+// BaseOSMgrStatus : for sending from baseosmgr
+type BaseOSMgrStatus struct {
+	BaseosUpdateCounter uint32 // Current BaseosUpdateCounter from baseosmgr
```

Comment:
Can we rename this field to CurrentRetryUpdateCounter (to be different than the config counter and also make it be consistent with these being named with "retry" elsewhere)

Reply:
Done